diff --git a/node_modules/expo-modules-autolinking/android/expo-gradle-plugin/expo-autolinking-plugin-shared/src/main/kotlin/expo/modules/plugin/AutolinkingCommandBuilder.kt b/node_modules/expo-modules-autolinking/android/expo-gradle-plugin/expo-autolinking-plugin-shared/src/main/kotlin/expo/modules/plugin/AutolinkingCommandBuilder.kt
index 5368156..1291eca 100644
--- a/node_modules/expo-modules-autolinking/android/expo-gradle-plugin/expo-autolinking-plugin-shared/src/main/kotlin/expo/modules/plugin/AutolinkingCommandBuilder.kt
+++ b/node_modules/expo-modules-autolinking/android/expo-gradle-plugin/expo-autolinking-plugin-shared/src/main/kotlin/expo/modules/plugin/AutolinkingCommandBuilder.kt
@@ -8,7 +8,7 @@ class AutolinkingCommandBuilder {
    * Command for finding and running `expo-modules-autolinking`.
    */
   private val baseCommand = listOf(
-    "node",
+    "/Users/jaimecriales/.nvm/versions/node/v22.22.3/bin/node",
     "--no-warnings",
     "--eval",
     "require('expo/bin/autolinking')",
diff --git a/node_modules/expo-modules-autolinking/android/expo-gradle-plugin/expo-autolinking-plugin/src/main/kotlin/expo/modules/plugin/ExpoAutolinkingPlugin.kt b/node_modules/expo-modules-autolinking/android/expo-gradle-plugin/expo-autolinking-plugin/src/main/kotlin/expo/modules/plugin/ExpoAutolinkingPlugin.kt
index 7029105..8b05d6d 100644
--- a/node_modules/expo-modules-autolinking/android/expo-gradle-plugin/expo-autolinking-plugin/src/main/kotlin/expo/modules/plugin/ExpoAutolinkingPlugin.kt
+++ b/node_modules/expo-modules-autolinking/android/expo-gradle-plugin/expo-autolinking-plugin/src/main/kotlin/expo/modules/plugin/ExpoAutolinkingPlugin.kt
@@ -53,7 +53,7 @@ open class ExpoAutolinkingPlugin : Plugin<Project> {
     project.providers.exec { spec ->
         spec.workingDir(nodeWorkingDir)
         spec.commandLine(
-            "node",
+            "/Users/jaimecriales/.nvm/versions/node/v22.22.3/bin/node",
             "--no-warnings",
             "--eval",
             "require('expo/bin/autolinking')",
diff --git a/node_modules/expo-modules-autolinking/android/expo-gradle-plugin/expo-autolinking-settings-plugin/src/main/kotlin/expo/modules/plugin/ExpoAutolinkingSettingsExtension.kt b/node_modules/expo-modules-autolinking/android/expo-gradle-plugin/expo-autolinking-settings-plugin/src/main/kotlin/expo/modules/plugin/ExpoAutolinkingSettingsExtension.kt
index 04601c1..d8ed1ab 100644
--- a/node_modules/expo-modules-autolinking/android/expo-gradle-plugin/expo-autolinking-settings-plugin/src/main/kotlin/expo/modules/plugin/ExpoAutolinkingSettingsExtension.kt
+++ b/node_modules/expo-modules-autolinking/android/expo-gradle-plugin/expo-autolinking-settings-plugin/src/main/kotlin/expo/modules/plugin/ExpoAutolinkingSettingsExtension.kt
@@ -52,7 +52,7 @@ open class ExpoAutolinkingSettingsExtension(
     File(
       settings.providers.exec { env ->
         env.workingDir(projectRoot)
-        env.commandLine("node", "--print", "require.resolve('@react-native/gradle-plugin/package.json', { paths: [require.resolve('react-native/package.json')] })")
+        env.commandLine("/Users/jaimecriales/.nvm/versions/node/v22.22.3/bin/node", "--print", "require.resolve('@react-native/gradle-plugin/package.json', { paths: [require.resolve('react-native/package.json')] })")
       }.standardOutput.asText.get().trim(),
     ).parentFile
   }
@@ -64,7 +64,7 @@ open class ExpoAutolinkingSettingsExtension(
     File(
       settings.providers.exec { env ->
         env.workingDir(projectRoot)
-        env.commandLine("node", "--print", "require.resolve('react-native/package.json')")
+        env.commandLine("/Users/jaimecriales/.nvm/versions/node/v22.22.3/bin/node", "--print", "require.resolve('react-native/package.json')")
       }.standardOutput.asText.get().trim(),
     ).parentFile
   }
diff --git a/node_modules/expo-modules-autolinking/android/expo-gradle-plugin/expo-autolinking-settings-plugin/src/main/kotlin/expo/modules/plugin/ExpoAutolinkingSettingsPlugin.kt b/node_modules/expo-modules-autolinking/android/expo-gradle-plugin/expo-autolinking-settings-plugin/src/main/kotlin/expo/modules/plugin/ExpoAutolinkingSettingsPlugin.kt
index d66e371..6121566 100644
--- a/node_modules/expo-modules-autolinking/android/expo-gradle-plugin/expo-autolinking-settings-plugin/src/main/kotlin/expo/modules/plugin/ExpoAutolinkingSettingsPlugin.kt
+++ b/node_modules/expo-modules-autolinking/android/expo-gradle-plugin/expo-autolinking-settings-plugin/src/main/kotlin/expo/modules/plugin/ExpoAutolinkingSettingsPlugin.kt
@@ -51,7 +51,7 @@ open class ExpoAutolinkingSettingsPlugin : Plugin<Settings> {
     val expoModulesAutolinkingPath =
       settings.providers.exec { env ->
         env.workingDir(settings.rootDir)
-        env.commandLine("node", "--print", "require.resolve('expo-modules-autolinking/package.json', { paths: [require.resolve('expo/package.json')] })")
+        env.commandLine("/Users/jaimecriales/.nvm/versions/node/v22.22.3/bin/node", "--print", "require.resolve('expo-modules-autolinking/package.json', { paths: [require.resolve('expo/package.json')] })")
       }.standardOutput.asText.get().trim()
 
     val expoAutolinkingDir = File(expoModulesAutolinkingPath).parentFile
