diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/src/main/kotlin/expo/modules/plugin/gradle/ExpoGradleHelperExtension.kt b/node_modules/expo-modules-core/expo-module-gradle-plugin/src/main/kotlin/expo/modules/plugin/gradle/ExpoGradleHelperExtension.kt
index 7c387ab..6b8f17e 100644
--- a/node_modules/expo-modules-core/expo-module-gradle-plugin/src/main/kotlin/expo/modules/plugin/gradle/ExpoGradleHelperExtension.kt
+++ b/node_modules/expo-modules-core/expo-module-gradle-plugin/src/main/kotlin/expo/modules/plugin/gradle/ExpoGradleHelperExtension.kt
@@ -43,7 +43,7 @@ open class ExpoGradleHelperExtension {
     reactNativeDir = reactNativeDirFromSource ?: File(
       project.providers.exec { env ->
         env.workingDir(project.rootDir)
-        env.commandLine("node", "--print", "require.resolve('react-native/package.json')")
+        env.commandLine("/Users/jaimecriales/.nvm/versions/node/v22.22.3/bin/node", "--print", "require.resolve('react-native/package.json')")
       }.standardOutput.asText.get().trim()
     ).parentFile
 
